Packs/iDefense/Integrations/iDefense_v2/iDefense_v2.py

```diff
@@ -0,0 +1,274 @@
+from typing import Union
+
+from CommonServerPython import *
+
+'''IMPORTS'''
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+'''CONSTANTS'''
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+class Client(BaseClient):
+
+    def __init__(self, input_url: str, api_key: str, verify_certificate: bool, proxy: bool):
+        base_url = urljoin(input_url, '/rest/threatindicator/v0')
+        headers = {
+            "Content-Type": "application/json",
+            'auth-token': api_key
+        }
+        super(Client, self).__init__(base_url=base_url,
+                                     verify=verify_certificate,
+                                     headers=headers,
+                                     proxy=proxy)
+
+    def perform_request(self, name: str, value: str):
```

Comment:
let's pick a more indicative name, e.g:
```suggestion
    def threat_indicator_search(self, name: str, value: str, data: dict = {}) -> dict:
```

Reply:
also, the name argument is more of a url_suffix no? please consider using a more fitting arg name.